solutions/usaco-944.mdx

```diff
@@ -0,0 +1,103 @@
+---
+id: usaco-944
+source: USACO Silver 2019 US Open
+title: Fence Planning
+author: Brad Ma
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_fenceplan_silver_open19.html)
+
+## Implementation
+**Time Complexity:** $\mathcal{O}(N + M)$
+
+<LanguageSection>
+  
+<JavaSection>
+
+```java
+import java.util.*;
+import java.io.*;
+
+public class FencePlanning {
+	static Cow[] cows;
+	static List<Integer>[] graph;
+	static boolean[] visited;
+	static int lowX = Integer.MAX_VALUE;
+	static int highX = Integer.MIN_VALUE;
+	static int lowY = Integer.MAX_VALUE;
+	static int highY = Integer.MIN_VALUE;
+
+	static class Cow { // class to keep track of the x and y
```

Comment:
ok when doing inline comments it's usually the convention to put two spaces between the comments and the code
```suggestion
	static class Cow {  // class to keep track of the x and y
```
keep in mind to this this for all the other inline comments

Reply:
there already is a space... why two?

Reply:
two's the convention